deployment/README.md

```diff
@@ -49,6 +49,6 @@ Secrets are managed with [Ansible Vault](docs.ansible.com/ansible/2.4/vault.html
 There is a tiny bridge vault file `vault-env` that bridges the `ANSIBLE_VAULT_PASSWORD` environment variable as Ansible vault password.
```

Comment:
nah @acidsennui we missed to change the var on this line.

Reply:
fixed.